e3db/tests/test_search_integration.py

```diff
@@ -13,12 +13,16 @@
 
 api_url = os.environ["DEFAULT_API_URL"]
 token = os.environ["REGISTRATION_TOKEN"]
+### Variables required for internal testing
+local_api_url = os.environ["LOCAL_API_URL"]
+local_boot_key = os.environ["LOCAL_BOOT_KEY"]
+local_boot_secret = os.environ["LOCAL_BOOT_SECRET"]
 
 class TestSearchIntegration():
     @classmethod
     def register_client(self):
         client1_public_key, client1_private_key = e3db.Client.generate_keypair()
-        client1_name = "client_{0}".format(binascii.hexlify(os.urandom(16)))
+        client1_name = "email_{0}@tozny.com".format(time.time())
```

Comment:
Any reason to go from random hex client names to time based ones?

Reply:
Oh is this because of the use of the internal endpoint for registration?

Reply:
Sorry I missed these: switched to time based instead of random hex, because it was more helpful to see clients were being created at X time for me; but no real preference for what goes here I just switch register to have parity with the internal_register method -- of which I couldn't figure out how to register a client w/ a local version of pds